include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_histogram.h

```diff
@@ -404,11 +403,11 @@ struct __histogram_general_private_global_atomics_submitter;
 template <typename... _KernelName>
 struct __histogram_general_private_global_atomics_submitter<__internal::__optional_kernel_name<_KernelName...>>
 {
-    template <typename _BackendTag, typename _Range1, typename _Range2, typename _BinHashMgr>
+    template <typename _Range1, typename _Range2, typename _BinHashMgr>
     sycl::event
-    operator()(_BackendTag, sycl::queue& __q, const sycl::event& __init_event,
-               ::std::uint16_t __min_iters_per_work_item, ::std::uint16_t __work_group_size, _Range1&& __input,
-               _Range2&& __bins, const _BinHashMgr& __binhash_manager)
+    operator()(sycl::queue& __q, const sycl::event& __init_event, ::std::uint16_t __min_iters_per_work_item,
+               ::std::uint16_t __work_group_size, _Range1&& __input, _Range2&& __bins,
```

Comment:
```suggestion
               std::uint16_t __work_group_size, _Range1&& __input, _Range2&& __bins,
```

Reply:
I see at least one more, I think you have a script that can fix all these, I wont call them out anymore.

Reply:
will fix

Reply:
fixed.